### DISCUSSION TITLE
Barracuda_libre-12.2050.3.release (Cortana Removal Update)

### DISCUSSION THREAD

**Discussion by coggy9** [coggy9/HKHacking#9] [General]
![image](https://user-images.githubusercontent.com/352207/115292920-42fd8800-a11c-11eb-9977-a83cf4e12de6.png)
After poking the OTA server a bit, I think I got the current update, but its not in a usable format. 

The file is only 6.3MB, way too small to be a full update, it's an incremental update to be applied over a stock system running 11.1842.0.
Binwalk only found compressed data and a PEM cert, and the compressed data can't be opened AFAIK.
![image](https://user-images.githubusercontent.com/352207/115293567-1bf38600-a11d-11eb-9197-428893d8e3dd.png)

[Here](https://b.catgirlsare.sexy/OF943wBX.) is the update package if anyone wants to take a look.

Spoofing the version as a lower FW (0.0.0) does not result in a full OTA link, OTA server just reports that there is no update.

Looks like the spoofed version is being correctly reported, if a Harman employee was confused. 😃

**Comment by j-a-lewis** [comment]
Have a dumb ignorant newbie question.  If I manage to get my Invokes updated to v12 but later decide to play around with the devices, will it be possible to go back and forth between v11 and v12 with the flashing tools Harman has provided or once we get v12 will restoring v11 be impossible based on what you folks know so far?  The reason I ask is that I don't think that I can learn enough about the Invoke to do anything useful before Harman's July 31st deadline for getting V12 OTA but it would be nice, having gotten the V12 update, to later have the option to go back and use at least one of my several Invokes as an experimental subject to learn new stuff.  I was wondering, too, how many flashings a device like the Invoke can typically stand up to before one has to worry about write fatigue - thousands of flashings possible?

**Reply by coggy9** [reply]
You can downgrade. Harman should be releasing an image of V12 sometime soon, so the OTA shouldn't be needed once they release that.

Not sure about the write fatigue. I've been trying to find a better way to flash only what changed in an image, because as of right now, everything gets erased and overwritten each time you flash an image.
